typescript/examples/by-use-case/solana-usdc-yield-deposit/README.md

```diff
@@ -0,0 +1,72 @@
+
```

Comment:
Could you add this to the main goat readme and the main typescript readme? In the by use case list.